fe1-web/network/MessageApi.ts

```diff
@@ -201,3 +203,32 @@ export function requestCloseRollCall(rollCallId: Number, attendees: PublicKey[])
   return publish(channelFromId(laoId), message);
 }
  */
+
+/** Sends a server query asking for creation of an Election with a given name (String),
+ *  an array of questions, a version (String), the current lao (String), the id, and the  creation,
+ *  start and end are also specified as a timestamp */
+export function requestCreateElection(
+  name: string,
+  version: string,
+  start: Timestamp,
+  end: Timestamp,
+  questions: Question[],
+): Promise<void> {
+  const time: Timestamp = Timestamp.EpochNow();
+  const currentLao: Lao = OpenedLaoStore.get();
+  const message = new SetupElection({
+    lao: currentLao.id.toString(),
+    id: Hash.fromStringArray(
+      EventTags.ELECTION, currentLao.id.toString(), currentLao.creation.toString(), name,
+    ),
+    name: name,
+    version: version,
+    created_at: time,
+    start_time: ((start > time) ? start : time),
+    end_time: end,
```

Comment:
You might have `start_time` > `end_time` with the protection you added at line 227. Might add such protection for `end_time` as well (cannot be smaller than `start_time`)

Reply:
Good point!

Reply:
I made it such that when a person does this it will put the end time 60 seconds after the start time. 
In the future it would be good to automatically block this behaviour in the UI.